src/harpstrata/types.ts

```diff
@@ -1,11 +1,13 @@
 import type { PitchMatrix } from './Pitch/types'
+import type { IsActiveComplex } from './IsActive'
```

Comment:
Consider renaming `IsActive` to `Activity`. This will probably be required in the future when we have different grades of activity.

Reply:
This is not a priority for this release